File: 6 Sorting/maxProductofThree/maxprodofthree.js
function solution(A) {
  A.sort((a, b) => (a - b));
  return Math.max(A[A.length - 1] * A[A.length - 2] * A[A.length - 3], A[A.length - 1] * A[0] * A[1]);
}
console.log(solution([-3, 1, 2, -2, 5, 6]));

File: 6 Sorting/maxProductofThree/maxProdOfThree.java

import java.util.Arrays;

class Solution {
    public int solution(int[] A) {
        Arrays.sort(A);
        int x = Math.max(A[A.length - 1] * A[A.length - 2] * A[A.length - 3], A[A.length - 1] * A[0] * A[1]);
        return x;
    }
}
